### PR TITLE
add zone support

### PR DESCRIPTION
**Pull request by PeckZeg**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,5 +10,5 @@ Thumbs.db
 logs
 node_modules
 dist
-lib
+# lib
```
Comment (Cap32) [inline .gitignore]: No need to commit `lib` to git

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "tiny-qiniu",
-  "version": "0.3.0",
+  "version": "0.3.1",
```
Comment (Cap32) [inline package.json]: No need to bump version, I will do it before I publish to `npm` 😃

**Comment by PeckZeg** [comment]
i will update and retry to create new pull request _(:зゝ∠)_

**Comment by Cap32** [comment]
Thanks 👍🏿

**Comment by Cap32** [comment]
`v0.4.0` published

**Pull request merged**